packages/connect-core/src/entities/Organization.ts

```diff
@@ -86,7 +86,11 @@ export default class Organization {
   ///////// APPS ///////////
   async apps(): Promise<Application[]> {
     this.checkConnected()
-    return this._connector.appsForOrg(this.address)
+    const apps = await this._connector.appsForOrg(this.address)
+    for (const app of apps) {
+      await app._init()
+    }
+    return apps
   }
 
   onApps(callback: Function): { unsubscribe: Function } {
```

Comment:
I'm not sure how to handle the `_init()` method for subscriptions. Ideas are welcome 😁